encord/utilities/storage/cloud_data_migration.py

```diff
@@ -0,0 +1,178 @@
+from __future__ import annotations
+
+import itertools
+from dataclasses import dataclass
+from typing import List, Optional, Union, cast
+from uuid import UUID
+
+import encord.orm.storage
+from encord import EncordUserClient
+from encord.http.bundle import BundlablePayload, Bundle, bundled_operation
+from encord.http.v2.api_client import ApiClient
+from encord.orm.base_dto import BaseDTO
+from encord.orm.cloud_integration import CloudIntegration
+from encord.storage import StorageItem
+
+STORAGE_CLOUD_DATA_MIGRATION_BUNDLE_LIMIT = 1000
+
+
+def update_storage_item_cloud_info(
+    user_client: EncordUserClient,
```

Comment:
Very styley: But currently no SDK functions take EncordUserClient as an argument and are instead methods on EncordUserClient.
So this isn't slightly different but not the end of the world

Reply:
As far as I understand, it is more like a "one off" call, rather than part of the usual sdk flow. Thus pushing it to the utilities makes sense to me. This way we have the functionality in the sdk, and do not pollute the UserClient interface too much.

Reply:
As far as I understand, it is more like a "one off" call, rather than part of the usual sdk flow. Thus pushing it to the utilities makes sense to me. This way we have the functionality in the sdk, and do not pollute the UserClient interface too much.